src/command.rs

```diff
@@ -1,13 +1,24 @@
+use crate::error::Result;
 use std::{
     path::Path,
     process::{Command, ExitStatus},
 };
-
-use crate::error::Result;
+use tracing::debug;
 
 /// Runs cargo command
 pub(crate) fn run_command(path: &Path, args: &[&str]) -> Result<ExitStatus> {
-    let mut command = Command::new("cargo").args(args).arg(path).spawn()?;
+    let output = Command::new("cargo").args(args).arg(path).output()?;
+
+    let stdout_str = String::from_utf8_lossy(&output.stdout).trim().to_string();
+    let stderr_str = String::from_utf8_lossy(&output.stderr).trim().to_string();
```

Comment:
We can perform this operation only if there is an effective error, so move this inside `if` below.  We can also remove `to_string` at the end and use directly `&str`. Does `output.stdout.to_string().trim()` work?